lightrec-private.h

```diff
@@ -49,6 +49,9 @@
 #define BLOCK_IS_DEAD		BIT(3)
 #define BLOCK_IS_MEMSET		BIT(4)
 
+/* USE A LARGER TYPE IF A BIT(8) FLAG IS ADDED IN FUTURE */
+#define block_flags_t u8
```

Comment:
typedef?

Reply:
actually could just get rid of it instead as nothing else uses it other than the struct definition

Reply:
Yep. And the comment can go away, if you add a BIT(8) later, the compiler will complain and we can update to a u16 then.